Subject: [PATCH] 1.7.2 header patch
---
Index: dojo.h
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/dojo.h b/dojo.h
--- a/dojo.h	(revision d6853af81e552ee4dd8609f36b4a37c4c1ff5867)
+++ b/dojo.h	(date 1758923193594)
@@ -24,7 +24,7 @@
 struct AttributeFilter;
 struct TokenBalance;
 struct TokenContract;
-enum ContractType;
+// enum ContractType;
 struct Contract;
 struct TokenTransfer;
 struct Provider;
